README.md

```diff
@@ -1,4 +1,4 @@
-# Cludo - Cloud Sudo
+# Cludo (Cloud Sudo) Client
```

Comment:
This title is also the first thing people see when they come to the repo.  Since the repo also contains the server, I don't want to confuse them.
```suggestion
# Cludo (Cloud Sudo)
```

Reply:
Fixed with a few more changes in https://github.com/superorbital/cludo/pull/70/commits/1f7211085de0743d3b539202ecb5c3296377fb3b